packages/flutter_tools/test/general.shard/ios/devices_test.dart

```diff
@@ -157,71 +165,129 @@ void main() {
 
     group('ios-deploy wrappers', () {
       const String appId = '789';
+      const String deviceId = 'device-123';
       IOSDevice device;
       IOSDeploy iosDeploy;
-      FullMockProcessManager mockProcessManager;
+      FakeProcessManager fakeProcessManager;
 
-      setUp(() {
-        mockProcessManager = FullMockProcessManager();
+      testWithoutContext('isAppInstalled() catches ProcessException from ios-deploy', () async {
+        final MockIOSApp mockApp = MockIOSApp();
+        when(mockApp.id).thenReturn(appId);
+        fakeProcessManager = FakeProcessManager.list(<FakeCommand>[
+          FakeCommand(
+            command: const <String>[
+              null,
```

Comment:
Why does this end up as `null`?

Reply:
Fixed by having `mockArtifacts` return `iosDeployPath`.